packages/odyssey-stylelint/package.json

```diff
@@ -21,7 +21,7 @@
   "devDependencies": {
     "jest": "^29.0.3",
     "jest-preset-stylelint": "^4.1.1",
-    "postcss": "^8.3.6",
+    "postcss": "^8.4.16",
     "postcss-scss": "^4.0.0"
```

Comment:
Updating TS, I needed to update `postcss` because the import types weren't working properly.